{
  "comments": [
    {
      "key": {
        "uuid": "b731cbd7_2577e2e8",
        "filename": "src/main/java/com/googlesource/gerrit/plugins/rabbitmq/session/type/AMQPSession.java",
        "patchSetId": 1
      },
      "lineNbr": 124,
      "author": {
        "id": 1011123
      },
      "writtenOn": "2018-08-30T05:27:50Z",
      "side": 1,
      "message": "Why would opening a connection result in this?  I would expect this kind of exception to occur when closing.",
      "range": {
        "startLine": 124,
        "startChar": 29,
        "endLine": 124,
        "endChar": 51
      },
      "revId": "6fba355ec5e76d8a3dc4baf17a184b001893fbe6",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153",
      "unresolved": true
    },
    {
      "key": {
        "uuid": "fd514c05_471219eb",
        "filename": "src/main/java/com/googlesource/gerrit/plugins/rabbitmq/session/type/AMQPSession.java",
        "patchSetId": 1
      },
      "lineNbr": 124,
      "author": {
        "id": 1014252
      },
      "writtenOn": "2018-08-30T06:33:45Z",
      "side": 1,
      "message": "Yes, the naming is threw me off a bit to. To make inconsistency worse it is thrown by AMQConnection#ensureIsOpen [1] which makes even less sense.\nI don\u0027t have the bandwidth to fix that inconsitency in the rabbitmq lib though.\n\n[1] https://github.com/rabbitmq/rabbitmq-java-client/blob/master/src/main/java/com/rabbitmq/client/impl/AMQConnection.java#L168",
      "parentUuid": "b731cbd7_2577e2e8",
      "range": {
        "startLine": 124,
        "startChar": 29,
        "endLine": 124,
        "endChar": 51
      },
      "revId": "6fba355ec5e76d8a3dc4baf17a184b001893fbe6",
      "serverId": "173816e5-2b9a-37c3-8a2e-48639d4f1153",
      "unresolved": true
    }
  ]
}